actions-server/package.json

```diff
@@ -18,10 +19,17 @@
     "node-fetch": "2.6.0"
   },
   "devDependencies": {
+    "@apollo/react-hooks": "^3.1.5",
+    "@graphql-codegen/cli": "^1.14.0",
+    "@graphql-codegen/typescript": "^1.14.0",
+    "@graphql-codegen/typescript-operations": "^1.14.0",
+    "@graphql-codegen/typescript-react-apollo": "^1.14.0",
     "@types/express": "^4.17.6",
     "@types/jsonwebtoken": "^8.5.0",
     "@types/node": "^14.0.1",
     "@types/node-fetch": "^2.5.7",
+    "@types/react": "^16.9.35",
+    "apollo-client": "^2.6.10",
```

Comment:
It's a little weird the server depends on some of these things but 🤷‍♂️

Reply:
i think a couple options here.. have two different codegen, which generate only the necessary code in each repo, or a new top level directory that just does codegen, and app/ & actions-server/ reference that.